src/xisfreader.rs

```diff
@@ -77,92 +77,105 @@ fn xisf_parse_xml(
+                                xisf_header.sample_format_bytes =
+                                    xisf_type_size(&xisf_header.sample_format);
                             }
-                        }
-                    } else if attr.name() == "compression" {
-                        // Parse compression. Format: "compression_algorithm:uncompressed-size"
-                        xisf_header.compression = attr.value().to_string();
-                        let split = xisf_header.compression.split(':');
-                        for (n, s) in split.enumerate() {
-                            match n {
-                                0 => xisf_header.compression_codec = s.to_string(),
-                                1 => xisf_header.compression_size = s.parse().unwrap(),
-                                _ => (),
+                            "colorSpace" => {
+                                // Parse space color
+                                xisf_header.color_space = attr.value().to_string();
+                            }
+                            "location" => {
+                                // Parse location. Format: "chan_size1:..:chan_size_n:n_channels" format
+                                xisf_header.location = attr.value().to_string();
+                                let split = xisf_header.location.split(':');
+                                for (n, s) in split.enumerate() {
+                                    println!("Location part: {}", s);
+                                    if n == 0 {
+                                        xisf_header.location_method = s.to_string();
+                                    } else if n == 1 {
+                                        xisf_header.location_start = s.parse().unwrap();
+                                    } else if n == 2 {
+                                        // location_length = image data size (compressed)
+                                        xisf_header.location_length = s.parse().unwrap();
+                                    }
+                                }
                             }
+                            "compression" => {
+                                // Parse compression. Format: "compression_algorithm:uncompressed-size"
+                                xisf_header.compression = attr.value().to_string();
+                                let split = xisf_header.compression.split(':');
+                                for (n, s) in split.enumerate() {
+                                    match n {
+                                        0 => xisf_header.compression_codec = s.to_string(),
+                                        1 => xisf_header.compression_size = s.parse().unwrap(),
+                                        _ => (),
+                                    }
+                                }
+                            }
+                            _ => {} //name => eprintln!("unknown attribute name {}", name),
```

Comment:
There are two options here:
```rust
_ => {}
```
That's the current one: don't do anything if the attribute is not one of the previous ones, or:
```rust
name => eprintln!("unknown attribute name {}", name),
```
Which will print a message in `stderr` saying that the attribute that was found is not known.

This is interesting to find things that might be wrong in the XML, but it's also annoying if we know there are more attributes but don't care about them.